### DISCUSSION TITLE
DBC-Wiki/en/staking-model/join-dbc-network

### DISCUSSION THREAD

**Discussion by giscus** [DeepBrainChain/DBC-Wiki#49] [Announcements]
# DBC-Wiki/en/staking-model/join-dbc-network

0. Recommended hardware RAM：8G; CPU：2 cores; Disk：100G; 1. Generate stash account （If you already have stash account, you can skip this） Reference: Generate new dbc account (gen...

https://deepbrainchain.github.io/DBC-Wiki/en/staking-model/join-dbc-network.html

**Comment by LPGOfficer** [comment]
Thanks for post this. 
It will be great to update with System and version to be used for better performance.
